webapp/shop/cred/views.py

```diff
@@ -526,10 +526,12 @@ def cred_your_exams(ua_contracts_api, trueability_api, **kwargs):
                         actions.extend(
                             [
                                 {
-                                    "text": "Continue exam"
-                                    if state
-                                    == RESERVATION_STATES["in_progress"]
-                                    else "Take exam",
+                                    "text": (
+                                        "Continue exam"
+                                        if state
+                                        == RESERVATION_STATES["in_progress"]
+                                        else "Take exam"
+                                    ),
```

Comment:
Everything on this file is Python linting. The linting Github action didn't pass if I didn't fix this.

Reply:
Thanks for fixing the Python linting issue!